package.json

```diff
@@ -5,16 +5,19 @@
   "main": "index.js",
   "scripts": {
     "build": "rm -rf dist && webpack",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "rm -rf dist && tsc && node dist/test/index.js"
   },
   "author": "adamv.stadler@gmail.com",
   "license": "ISC",
   "dependencies": {
     "@slack/web-api": "^5.13.0",
+    "discord.js": "^12.5.0",
+    "ffmpeg-static": "^4.2.7",
```

Comment:
What's ffmpeg for? Co-dependency for discord or something of the such?

Reply:
a discord dependency that wasn't properly included in the discord package